### ISSUE TITLE
An error occured during the Numverify scan

### ISSUE THREAD

**Issue by MoldyCheesecake** [sundowndev/PhoneInfoga#338]
**Describe the bug**
When scanning some phone numbers, the following error occur :

> [i] Running Numverify.com scan...                                                                     
> [!] An error occurred                                                                                 
> [!] EOF 

This only happens when trying to scan two specific numbers, I already achieved to scan many more without any issue

**To Reproduce**
./phoneinfoga scan -n phoneNumber

**Expected behavior**
I didn't expected an error during the Numverify scan, which usually goes well

**Screenshots (optional)**
![PhoneInfoga](https://user-images.githubusercontent.com/65112449/81496847-f75f1f00-92ba-11ea-9333-0af604655c68.png)

**Desktop (please complete the following information):**
 - OS: [Kali Linux 2020.1]
 - PhoneInfoga exact version : 2.0.7
 - Go exact version (1.14.2)

**Additional context**
Even though the numbers on the screenshot seems similar, their two last digits are different.

**Comment by sundowndev** [comment]
Hi, Numverify answered an empty response, causing a EOF (End Of File) error. I'm working on a better handling of this error.

**Comment by sundowndev** [comment]
Patched in [v2.0.11](https://github.com/sundowndev/PhoneInfoga/releases/tag/v2.0.11). Since this is an issue with Numverify's API, the error will not disappear but you'll be able to ignore it and continue scanning.

```
[i] Running Numverify.com scan...
[!] An error occurred: EOF
[!] It may be Numverify not finding information about that number, or the number being invalid.
[i] Generating Google search dork requests...
[i] Social media footprints
[+] Link: https://www.google.com/search?q=site%3Afacebook...
...
```
